### ISSUE TITLE
ActiveJob compatibility

### ISSUE THREAD

**Issue by skydan** [ondrejbartas/sidekiq-cron#33]
I'm using sidekiq as backend for AJ. All works fine except enqueueing: 
```
2015-03-19T20:20:00.916Z 9752 TID-owfr4v32w WARN: {"retry"=>true, "queue"=>"high", "class"=>"ParseRepoJob", "args"=>["self"], "jid"=>"8759e3380b726d242347f1d0", "enqueued_at"=>1426796400.914447}
2015-03-19T20:20:00.916Z 9752 TID-owfrvkf88 INFO: Cron Jobs - add job with name: parse_self_repo
2015-03-19T20:20:00.916Z 9752 TID-owfr4v32w WARN: undefined method `jid=' for #<ParseRepoJob:0x007fc81160b0b8>
```
Is there a easy way to fix that?

**Comment by philipgiuliani** [comment]
Would be amazing to get this!

**Comment by EvanMcEwen** [comment]
Add my vote to the mix

**Comment by ondrejbartas** [comment]
Hi, i will try to fix this. BTW i havent used active jobs. So this will be first time for me :dancer:

**Comment by philipgiuliani** [comment]
Nice to hear that! Good luck :)

**Comment by EvanMcEwen** [comment]
Yesssss thank you for looking into this :+1:

**Comment by lgn21st** [comment]
@ondrejbartas thanks and good luck!

**Comment by ondrejbartas** [comment]
pls can you try to use sidekiq-cron with active jobs in your projects?
use

`gem 'sidekiq-cron', git: 'https://github.com/ondrejbartas/sidekiq-cron.git', branch: 'active-job-support'`

and let me know if it is working.

**Comment by lgn21st** [comment]
@ondrejbartas It works as expectation, thank you very much!

**Comment by ondrejbartas** [comment]
I will update test suite and then release new version.

**Comment by ondrejbartas** [comment]
released version 0.3.0
